org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java

```diff
@@ -33,7 +33,9 @@
   private static final int CHECK_THREAD_INTERVAL_MS = 500;
   private static final int TIMEOUT_WARNING_MS = 20000;
   private static final int TIMEOUT_CLEANUP_BUFFER_MS = 5000;
+  private static final CachedResources cachedResources = new CachedResources();
```

Comment:
Just to check my understanding -- in order for something to be re-used across Lambda invocations, does it have to be a static _in this class_?  Or can statics declared elsewhere also be reused?

Reply:
it does have to be static in this class specifically